src/protocol/prss.rs

```diff
@@ -4,11 +4,16 @@ use crate::rand::{CryptoRng, RngCore};
 use crate::secret_sharing::Replicated;
 use crate::sync::{Arc, Mutex};
 
+#[cfg(not(feature = "turn-off-prss"))]
```

Comment:
Rather than puncturing this code, it might be easier to build a parallel file that you include like this:

```rust
#[cfg(feature = "no-prss")]
mod prss_zero;
#[cfg(feature = "no-prss")]
pub use prss_zero::{...};

#[cfg(not(feature = "no-prss"))]
mod prss;
#[cfg(not(feature = "no-prss"))]
pub use prss::{...};
```

There will be a bit of duplication, but I think that this would be easier to read and manage.

Reply:
makes sense. let me do this